specification/sql/resource-manager/Microsoft.Sql/stable/2014-04-01/disasterRecoveryConfigurations.json

```diff
@@ -0,0 +1,405 @@
+        },
+        "parameters": [
+          {
+            "$ref": "#/parameters/ApiVersionParameter"
+          },
+          {
+            "$ref": "#/parameters/SubscriptionIdParameter"
+          },
+          {
+            "$ref": "#/parameters/ResourceGroupParameter"
+          },
+          {
+            "$ref": "#/parameters/ServerNameParameter"
+          },
+          {
+            "name": "disasterRecoveryConfigurationName",
+            "in": "path",
+            "required": true,
+            "type": "string",
+            "description": "The name of the disaster recovery configuration."
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "OK",
+            "schema": {
+              "$ref": "#/definitions/DisasterRecoveryConfiguration"
+            }
+          }
+        }
+      }
+    },
+    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.Sql/servers/{serverName}/disasterRecoveryConfiguration/{disasterRecoveryConfigurationName}/failover": {
+      "post": {
+        "tags": [
+          "DisasterRecoveryConfigurations"
+        ],
+        "operationId": "DisasterRecoveryConfigurations_Failover",
+        "description": "Which disaster recovery configuration is being failed-over.",
+        "x-ms-examples": {
+          "Failover a disaster recovery configuration": { "$ref": "./examples/DisasterRecoveryConfigurationFailover.json" }
+        },
+        "parameters": [
+          {
+            "$ref": "#/parameters/ApiVersionParameter"
+          },
+          {
+            "$ref": "#/parameters/SubscriptionIdParameter"
+          },
+          {
+            "$ref": "#/parameters/ResourceGroupParameter"
+          },
+          {
+            "$ref": "#/parameters/ServerNameParameter"
+          },
+          {
+            "name": "disasterRecoveryConfigurationName",
+            "in": "path",
+            "required": true,
+            "type": "string",
+            "description": "The name of the disaster recovery configuration to failover."
+          }
+        ],
+        "responses": {
+          "204": {
+            "description": "No Content"
+          },
+          "202": {
+            "description": "Accepted"
+          }
+        },
+        "x-ms-long-running-operation": true
+      }
+    },
+    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.Sql/servers/{serverName}/disasterRecoveryConfiguration/{disasterRecoveryConfigurationName}/forceFailoverAllowDataLoss": {
+      "post": {
+        "tags": [
+          "DisasterRecoveryConfigurations"
+        ],
+        "operationId": "DisasterRecoveryConfigurations_FailoverAllowDataLoss",
+        "description": "Which disaster recovery configuration is being failed-over forcefully. This operation might result in data loss.",
```

Comment:
Is this description correct? (it's starting with `which`)

Reply:
Replaced the description